docs/source/architecture/poet.rst

```diff
@@ -419,7 +419,7 @@ The server side of the validator runs the following sign-up procedure:
    a. Pass sign-up certificate of new participant (OPK, EPID Pseudonym, PPK,
       current :math:`WaitCertId_{n}` to upper layers for registration in EndPoint
       registry.
-   #. Goto 1.
+   #. Goto 1，namely waiting for a join request.
```

Comment:
How about `s/for a join/for a new join/`?

Reply:
I prefer 'for a new join'